v3/src/utilities/tokened-string.ts

```diff
@@ -0,0 +1,22 @@
+/* This file is a temporary measure until we implement a more robust localized string assembly system */
```

Comment:
I imagine we'll have something else doing this job (replacing words in a string template with variable number of identical tokens)

Reply:
Ok,  but -- unless I misunderstand  -- for now I'll leave it in as it is doing the job that may be done down the road by something else.

Reply:
My point is that the _something else_ is already on `main`.